sklearn/mixture/dpgmm.py

```diff
@@ -501,10 +501,10 @@ def fit(self, X):
             List of n_features-dimensional data points.  Each row
             corresponds to a single data point.
         """
-        self.random_state = check_random_state(self.random_state)
+        self.random_state_ = check_random_state(self.random_state)
```

Comment:
do you think that will be different from a ``None`` seed? That is the default parameter and so we do test for it (well after this PR we do)